xgcm/sgrid.py

```diff
@@ -0,0 +1,230 @@
+from collections import OrderedDict
```

Comment:
Dicts now  preserve insertion order by default so this should be unnecessary.

Reply:
Thanks!
This is the same approach as used in the `grid.py` and `comodo.py`, so I am tempted to perhaps leave it for now in favor of a larger refactor later.

Reply:
Fine with this here. Could you make a short issue to track this? That would be very helpful.